internal/sms-gateway/modules/push/service.go

```
@@ -102,10 +140,45 @@
 		s.logger.Error("Can't send messages", zap.Error(err))
+		return
+	}
+
+	for token, sendErr := range errs {
+		s.logger.Error("Can't send message", zap.Error(sendErr), zap.String("token", token))
+
+		wrapper := targets[token]
+		wrapper.retries++
+
+		if wrapper.retries >= maxRetries {
+			s.blacklist.Set(token, struct{}{})
```

golangci-lint (1.64.8)

Error return value of `s.blacklist.Set` is not checked

(errcheck)